### DISCUSSION TITLE
[Bug] Publishing failed - no error explanation (also not in console)

### DISCUSSION THREAD

**Discussion by codewulf0** [webstudio-is/webstudio-community#151] [Q&A] [answered]
I want to publish a site but get an error, without any indication what the error is.
![Scherm­afbeelding 2024-06-06 om 10 15 18](https://github.com/webstudio-is/webstudio-community/assets/85789160/46c8e99a-8720-4e05-8af3-26646687e646)

**Comment by kof** [comment]
Which project is it?

**Reply by kof** [reply]
we see one build failing, just to make sure its yours

**Reply by codewulf0** [reply]
Hi Kof, it's this one: https://apps.webstudio.is/builder/23c91dce-26bb-4f47-a6e3-6e09368ca5f5?authToken=b109c51f-e881-4b22-81f6-ab0772de58c3&mode=preview

I'm also talking to Oleg on Discord for an issue related to something on this page. I bet its related :)

**Comment by kof** [comment, marked as answer]
fixed
